apps/src/templates/teacherHomepage/GradientNavCard.story.jsx

```diff
@@ -18,7 +10,11 @@ export default storybook => {
         description: `This is an example navigation card with stub data.`,
         story: () => (
           <GradientNavCard
-            cardData={EXAMPLE_CARD_DATA}
+            title= "Teacher Community"
+            description="Ask questions about curriculum, share ideas from your lessons, and get help from other teachers"
+            image="../../static/navcard-placeholder.png"
+            buttonText= "Connect Today"
+            link= "link to teacher community"
```

Comment:
I get this error: Refused to display 'https://code.org/educate/community' in a frame because an ancestor violates the following Content Security Policy directive: "frame-ancestors 'self' http://*.disney.com http://*.diznee.net code.org studio.code.org", when I put a link in for the Storybook story.  Right now I'm working on including these GradientNavCards on the TeacherHomepage component, rendering that big component to the /teacher page, and from there I should be able to check links. It's coming...